rules/keep-tasks-handy-for-calls/rule.md

```diff
@@ -0,0 +1,32 @@
+---
+type: rule
+title: Do you always keep your task list handy for when the boss calls?
+uri: keep-tasks-handy-for-calls
+authors:
+  - title: Tanya Leahy
+    url: https://ssw.com.au/people/tanya-leahy/
+created: 2024-05-22T13:55:00.000Z
+guid: 8f304a6b-8345-4e3b-ac6f-259e028bb0c2
+
+---
+
+In today’s busy work environment, it is crucial to stay organized and be ready to update your boss on your tasks at any moment. Using Microsoft Loop in a Teams Tab can help you manage your tasks effectively.
+
+<!--endintro-->
+::: bad  
+![Figure: Bad example - Unorganised tasks written in a notebook](rules/keep-tasks-handy-for-calls/notebook-task-list.png)
+:::
+
+::: good  
+![Figure: Good example - Organized task list using Loop in a Teams tab](rules/keep-tasks-handy-for-calls/loop-tasks-in-teams.png)
+:::
+
+### Steps to Use Loop in a Teams Tab
```

Comment:
Hi @Tanya-Leahy 

1. The word "use" should not be capitalized as per https://www.ssw.com.au/rules/how-to-capitalize-titles/

Reply:
Done